server/models/message.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*Contains the User struct, as well as any methods*/
+package models
+
+//All strings for now, type will be changed as project develops
+type Message struct {
+	Name    string
```

Comment:
I think we also need the timestamp, and maybe we could use that as a key, for messages, so they get ordered.

Reply:
and also senderID